radiomics/imageoperations.py

```diff
@@ -248,7 +239,117 @@ def applyThreshold(inputImage, lowerThreshold, upperThreshold, insideValue=None,
   return tif.Execute(inputImage)
 
 
-def swt3(inputImage, wavelet="coif1", level=1, start_level=0):
+def applyFilterOriginal(inputImage, **kwargs):
```

Comment:
Either way is fine with me, I just wanted to make sure Hugo is happy with the conventions, and they are consistent throughout the toolkit and documentation.